fladgejt/webui/__init__.py

```diff
@@ -20,4 +20,4 @@ def check_connection(self):
         check_connection(self.context)
 
     def logout(self):
-        self.context.request_html('/ais/logout.do')
+        self.context.request_html('/ais/logoutCosign.do')
```

Comment:
@Adman it would be interesting to see whether we can get access to the `server` settings here.